corehq/apps/app_manager/views/releases.py

```diff
@@ -97,8 +98,11 @@ def paginate_releases(request, domain, app_id):
         app_es = app_es.is_released()
     if build_comment:
         app_es = app_es.build_comment(build_comment)
-    apps = app_es.run()
-    saved_apps = [SavedAppBuild.wrap(app).to_saved_build_json(timezone) for app in apps.hits]
+    app_ids = app_es.get_ids()
+    apps = get_docs(Application.get_db(), app_ids)
```

Comment:
I think if you have now fixed the `wrap` method to not save, we could revert this to only pull from ES, since all the information is there anyway, and it will stop a second round trip request. 
You could also exclude the translations key from being returned by ES using the [`source` field ](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-source-filtering.html)

We have utilities around that here:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/es_query.py#L369-L383

Reply:
I also like Ethan's idea, and you could probably just pull the source fields that we actually need for the view from ES.

Reply:
Though about it and shared my concern in the PR description,

>I initially was trying to work towards a wrapper which would only returns things in the ajax request that the releases page actually needs but that looked like a big fix that might break things. So just kept it the opposite for now to simply avoid things that are not used by the releases page for sure, starting with translations.

Reply:
@esoergel fyi ^

Reply:
also, from the PR description

> This PR avoid update during the wrap when fetching releases, so should we revert to ES? But it still looks like the right change considering couch model wrapping ES data.

I am unable to wrap my head around this approach where we take data from ES and then wrap it as a couch model, so i am not sure about reverting the change of DB(ES-> Couch).